geometry_msgs/Point rel_position
float64 width
float64 height
float64 box_angle
